tests/unit/models/test_sbom.py

```diff
@@ -559,12 +559,14 @@ def test_cyclonedx_sbom_can_be_converted_to_spdx_and_back_without_loosing_any_da
     }
 )
 
-DEFAULT_ROOT_RELATION = {
-    "spdxElementId": "SPDXRef-DOCUMENT",
-    "comment": "",
-    "relatedSpdxElement": "SPDXRef-DocumentRoot-File-",
-    "relationshipType": "DESCRIBES",
-}
+DEFAULT_ROOT_RELATION = SPDXRelation(
+    **{
+        "spdxElementId": "SPDXRef-DOCUMENT",
+        "comment": "",
+        "relatedSpdxElement": "SPDXRef-DocumentRoot-File-",
+        "relationshipType": "DESCRIBES",
+    }
+)
```

Comment:
nitpick: I'm wondering if `SPDXRelation.from_dict()` wouldn't be more readable, but that's just me, I missed the unpack at first glance, feel free to ignore.

Reply:
`SPDXRelation` does not have `from_dict` at the moment, I'll keep it as expansion for now.

Reply:
I meant we could add one :) , it's simple, out of scope though.